modules/homer/maketagdirectory/meta.yml

```
@@ -0,0 +1,41 @@
+name: homer_maketagdirectory
+description: Create a tag directory with the HOMER suite
+keywords:
+  - peaks
+  - bed
+tools:
+  - homer:
+      description: |
+        HOMER (Hypergeometric Optimization of Motif EnRichment) is a suite of tools for Motif Discovery and next-gen sequencing analysis.
+      documentation: http://homer.ucsd.edu/homer/
+      doi: 10.1016/j.molcel.2010.05.004.
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - peaks:
```

```suggestion
  - bed:
```

Just to be in line with what is in the `main.nf` file